client/lib/preferences-helper/PreferenceList.js

```diff
@@ -0,0 +1,69 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { partial, isEmpty } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import { getAllRemotePreferences } from 'state/preferences/selectors';
+import { savePreference } from 'state/preferences/actions';
+
+class PreferenceList extends Component {
+	render() {
+		const { preferences, translate, unsetPreference } = this.props;
+		return (
+			<div>
+				<a
```

Comment:
We should probably include a `<QueryPreferences/>` around here as we cannot assume that they have been loaded already.